test/modules/engin/pkg_module_res.skipif

```diff
@@ -0,0 +1,3 @@
+# this test requires curl.h which isn't installed in nightly cygwin
```

Comment:
Is there a way to just check for the header instead of hardcoding that these machines lack it?

Reply:
Yes but I wasn't sure it was worth the effort for this particular test, which used to be a compilation future. It doesn't actually use curl in a meaningful way and I also considered changing the test to not use curl at all.

Reply:
I'd be in favor of changing the test to not use curl.

Reply:
As you wish!